specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/examples/identity/GetLedgerIdentity.json

```diff
@@ -2,7 +2,7 @@
   "parameters": {
     "api-version": "2022-05-13",
     "ledgerId": "contoso-ledger-12345",
-    "identityServiceUri": "identity.accledger.azure.com"
+    "certificateClientEndpoint": "identity.accledger.azure.com"
```

Comment:
Let's make this `identity.confidential-ledger.core.azure.com` which is the official endpoint now